migrations/migrationHelpers.js

```diff
@@ -90,7 +91,11 @@ const deployContract = async (
     config.contract = 'contracts/tokenization/RAZOR.sol:RAZOR';
   }
 
-  await hre.run('verify:verify', config);
+  try {
+    await hre.run('verify:verify', config);
+  } catch (err) {
+    console.error('Etherscan verification failed');
```

Comment:
Log the error as well here.

Reply:
resolved https://github.com/razor-network/contracts/pull/579/commits/323a53f4abccbe76f5a7a48631eed413176bec18